troncos/contrib/asgi/logging/middleware.py

```diff
@@ -92,9 +92,9 @@ def __init__(
         logger_name: str | None = None,
     ) -> None:
         self._app = app
-        ln = logger_name or "asgi"
-        self._access = get_logger(f"troncos.{ln}.access")
-        self._error = get_logger(f"troncos.{ln}.error")
+        ln = logger_name or "troncos.asgi"
+        self._access = get_logger(f"{ln}.access")
+        self._error = get_logger(f"{ln}.error")
```

Comment:
Pure personal preference. Hate seeing `troncos.<servicename>.access` in the logs.

Reply:
👍  It could be nice to use some kind of shared prefix, that makes easier to find "core operation" logs easier. You don't have to know the app in detail to find it. But i'm in.

Reply:
I think is easy enough to search for `*.access`

Reply:
Yes, `*.access|*.error`